README.md

```diff
@@ -167,6 +167,13 @@ You find minimal example apps that illustrate the commands at [./examples](./exa
 
     ``binge nuke fileName`` removes the specified file from all local packages and the root package, analogous to ``binge copy``.
 
+* ``remove dependencyName``
```

Comment:
let's use the commander syntax so:
<cliArgument> -> mandatory 
[clientArgument] -> optional
<cliArgument...> -> mandatory list argument
[clientArgument...] -> optional list argument

So in this case it would be remove <dependencyName...>

What do you think?